db/migrate/20180822005716_create_pieces.rb

```diff
@@ -0,0 +1,17 @@
+class CreatePieces < ActiveRecord::Migration[5.0]
+  def change
+    create_table :pieces do |t|
+      t.string :name
+      t.integer :game_id
+      t.integer :user_id
+      t.string :moved_from_x
```

Comment:
Could you make these 4 columns into 2 columns? Just a x_position and y_position?

Reply:
I will make that change. I agree that there is no need to store the previous location of a piece in the DB. If we want to implement a move history, this can be done other ways.